tests/handlers/permissions-request.spec.ts

```diff
@@ -57,6 +57,25 @@ export function testPermissionsRequestHandler(): void {
         await dwn.close();
       });
 
+      it('should process a PermissionsRequest with EventStream not set', async () => {
```

Comment:
For my understanding: what is this test attempting to cover? I didn't think `EventStream` non-existence could inadvertently cause issue to handling of `PermissionsRequest`. What am I missing?

Reply:
Ah, looks like a code coverage thing, just noticed you added it for all Interface Methods. What line is CC complaining about...?!